### PR TITLE
add template7@1.1.3 w/ git auto-update

### LINKED ISSUE #9480: [Request] Add Template7
Library name: Template7
Git repository url: https://github.com/nolimits4web/template7
npm package url(optional): 
License(s): MIT
Official homepage: http://idangero.us/template7/
Wanna say something? Leave message here:

### PR DESCRIPTION
**Pull request by qwe19272375**
Pull request for issue: #9480 
Related issue(s): #9414 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/nolimits4web/template7
 * Official website (optional, not the repository): http://idangero.us/template7/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 318
   - Count of watchers: 26
   - Count of forks: 77
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9480, cc @nolimits4web #9414

### CODE REVIEW

**Comment by qwe19272375** [comment]
@nolsherry Please help me review this PR, thanks!

File: ajax/libs/template7/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "iphone",
+    "ipad",
+    "phonegap",
+    "framework7",
+    "handlebars",
+    "mustache"
+  ],
+  "author": "Vladimir Kharlampidi",
+  "license": "MIT",
+  "homepage": "http://www.idangero.us/template7/",
```
Comment (nolsherry) [inline ajax/libs/template7/package.json]: @qwe19272375 please modify `homepage` url to `https://idangero.us/template7/`

**Comment by qwe19272375** [comment]
`homepage` url modified. Please help me review it, thanks!

**Comment by nolsherry** [comment]
LGTM, ping @cdnjs/intern2

**Comment by kennynaoh** [comment]
@qwe19272375 Please rebase to the latest master branch, thanks!

**Comment by kennynaoh** [comment]
Next time you need to tag mentor after you fix the problem, thanks!

**Comment by kennynaoh** [comment]
LGTM

**Pull request merged**